webapp/src/main/webapp/js/individual/manageLabelsForIndividual.js

```diff
@@ -54,37 +54,37 @@ var manageLabels = {
         	this.onShowAddForm();
 
         	//Also make sure the save button is enabled in case there is a value selected for the drop down
-        	if(this.labelLanguage.val() != "") {
-        		disableSubmit = false;
-        	}
+        	// if(this.labelLanguage.val() != "") {
```

Comment:
Instead of commenting out, please remove the code. We can always find it in the history.

Reply:
Okay, i will remove this file and yes, the earlier fix can then be removed.
I will remove the JS-code too